site/content/blogs/release-v125.en.md

```diff
@@ -31,6 +31,98 @@ Copilot v1.25 brings several new features and improvements:
+
+Addons are additional AWS resources that are not integrated in Copilot by default - for example, DynamoDB, RDS, etc. 
+Environment addons are additional resources managed on the environment level. Their lifecycle will be tied
+to your environment: you will run `copilot env deploy` to create or update your environment addons; 
+when you run `copilot env delete` against an environment, Copilot will try to delete its addons as well.
+
+If you are already familiar with workload addons, then good news - 
+the experience of managing environment addons is pretty similar.
+
+#### Getting Started
+##### Step 1: Model additional AWS resources with CloudFormation
+Today, addons only support modelling using CloudFormation. For environment addons, you must:
+
+1. Have `App` and `Env` in the `Parameters` section.
+2. Include at least one `Resource`.
+
+[Here](TODO: link) is an example CloudFormation template that you can use to experiment.
+
+##### Step 2: Store the CFN template under `copilot/environments/addons`
+
+If you have run `copilot env init`, you should already have the folder `copilot/environments` in your workspace.
+If you haven't, it's recommended to do so now - you will need it sooner or later.
+
+At the end, your workspace structure may look like this:
+```
+copilot/
+├── environments/ 
+│   ├── addons/  
+│   │     ├── appmesh.yml         
+│   │     └── ddb.yml      # <- You can have multiple addons.  
+│   ├── test/ 
+│   │     └─── manifest.yml 
+│   └── dev/ 
+│         └── manifest.yml
+└── web
+    ├── addons/
+    │     └── s3.yml       # <- A workload addon template.
+    └─── manifest.yml 
+```
+
+##### Step 3: Run `copilot env deploy`
+
+After you run `copilot env deploy`, Copilot will scan through the `addons` folder to look for addons template. 
+If it is able to find any, it will deploy the templates along with the environment. 
+
+##### (Optional) Step 4: Verify the deployment
+
+You can verify the deployment by going to the [AWS CloudFormation console](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks) 
+in your region. You should be able to find a stack named `[app]-[env]-AddonsStack-[random string]`. This is a nested stack
+created under your environment stack, named `[app]-[env]`.
+
+### Feature Parity With Workload Addons
+Environment addons is shipped with all existing features available for workload addons. This means that:
+
+1. You can refer to customized `Parameters` in addition to the required `App` and `Env`.
+2. In your templates, you can reference local paths. Copilot will upload those local files and replace the relevant
+resource properties with the uploaded S3 location.
+
+Read [here](TODO: link) for more!
+
+### Other Considerations
+All environments (in the example above, both "test" and "dev") will share the same addon templates. 
+Just like today’s workload-level addons, any environment-specific configuration should be specified inside the 
```

Comment:
Should we emphasize that they won't be able to deploy two different envs with addons without making some env-specific config bc there will be a resource duplication?

Reply:
> resource duplication

As long as they don't hard code the resource name, CFN will auto generate a resource name based on the stack name, which is namespaced by app and env. But yes there is the possibility of resource duplication when they do hardcode the name

I want to avoid including too much information in the blog post though. Do you think it's acceptable if I mention it in the doc?